package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "clean": "rm -rf build && mkdir build",
     "build:server": "babel -d ./build ./server -s",
-    "build:client": "webpack -p --config ./webpack.production.config.js",
+    "build:client": "webpack -p --progress --colors --watch  --config ./webpack.production.config.js",
```

Comment:
This is the webpack production build. You wanna add these options to the webpack config because this command is evoked only for production builds up at heroku. We don't want webpack to watch the production build (it's a waste of resources).